src/pages/Login.js

```diff
@@ -89,7 +89,7 @@ export default () => {
             Continue with GitHub
           </StyledButton>
         </ButtonContainer>
-        <A href="/">Skip For Now</A>
+        <A href="/">Return to Portal</A>
```

Comment:
can we do a ternary on `copyText.hackathonName` for this?

Reply:
isn't the Skip for Now for nwHacks?

Reply:
We just decided to change the copy, both will be the same

Reply:
no there is only one login.js page that changes appearance based on them, so both HC and nw have this link rn 

this gives all users an option to go back to dashboard if they decide they dont' want to proceed 

if you want to remove this link here (I admit I was a bit presumptuous when i made it) i'm cool with it, just let me know if Logout should redirect to Login page or to home page

Reply:
+ ternary probably too complicated once we run 4 hackathons..